array_api_tests/test_signatures.py

```diff
@@ -32,8 +32,7 @@ def squeeze(x, /, axis):
 from . import dtype_helpers as dh
 from . import hypothesis_helpers as hh
 from . import xps
-from ._array_module import _UndefinedStub
-from ._array_module import mod as xp
+from ._array_module import _UndefinedStub, mod as xp
```

Comment:
Hmm, I'm guess you prefer this import style. I've been using `black` and `isort` locally for the newer files I introduced, so by virtue prefer how it is already :sweat_smile:

Reply:
It does seem weird to put these on different lines, but I didn't realize that this was left over from other changes that I originally made in this file, so we can revert it.